types/index.d.ts

```diff
@@ -839,6 +839,16 @@ declare namespace RamdaAdjunct {
          */
         rejectP<T>(value?: T): Promise<T>;
 
+        /**
+         * Composable shortcut for `setTimeout(Promise)`.
```

Comment:
The description here needs to match the to the original function description here: https://github.com/char0n/ramda-adjunct/pull/1107/files#diff-c90c9d832e4d7336f62bd494cc38e07cR5

Pls drop the `Composable shortcut for setTimeout(Promise).` sentence

Reply:
Sure